### PR TITLE
CMS high-mass Drell-Yan at 13 TeV

### PR DESCRIPTION
**Pull request by voisey**

### CODE REVIEW

File: buildmaster/src/buildmaster.cc [hunk 1]
```diff
@@ -223,18 +223,22 @@ vector<std::unique_ptr<CommonData>> InitCommonData()
...
+target.emplace_back(new CMSWMASY47FBFilter())
+target.emplace_back(new CMSWMU8TEVFilter())
+target.emplace_back(new CMSZDIFF12Filter())
+target.emplace_back(new CMS_1JET_8TEVFilter())
+target.emplace_back(new CMS_2JET_3D_8TEVFilter())
+target.emplace_back(new CMS_2JET_7TEVFilter())
+target.emplace_back(new CMS_HMDY_13TEVFilter())
+target.emplace_back(new CMS_HMDY_DE_13TEVFilter())
+target.emplace_back(new CMS_HMDY_DM_13TEVFilter())
+target.emplace_back(new CMS_ZCHARM_DIFF_UNNORM_8TEVFilter())
```
Comment (Zaharid) [inline buildmaster/src/buildmaster.cc:240]:
```
  target.emplace_back(new CMS1JET276TEVFilter());
  target.emplace_back(new CMSDY2D11Filter());
  target.emplace_back(new CMSDY2D12Filter());
  target.emplace_back(new CMSJETS11Filter());
  target.emplace_back(new CMSWEASY840PBFilter());
  target.emplace_back(new CMSWMASY47FBFilter());
  target.emplace_back(new CMSWMU8TEVFilter());
  target.emplace_back(new CMSZDIFF12Filter());
  target.emplace_back(new CMS_1JET_8TEVFilter());
  target.emplace_back(new CMS_2JET_3D_8TEVFilter());
  target.emplace_back(new CMS_2JET_7TEVFilter());
  target.emplace_back(new CMS_HMDY_13TEVFilter());
  target.emplace_back(new CMS_HMDY_DE_13TEVFilter());
  target.emplace_back(new CMS_HMDY_DM_13TEVFilter());
  target.emplace_back(new CMS_ZCHARM_DIFF_UNNORM_8TEVFilter());
```

Comment (Zaharid) [inline buildmaster/src/buildmaster.cc:240]:
```suggestion
  target.emplace_back(new CMS1JET276TEVFilter());
  target.emplace_back(new CMSDY2D11Filter());
  target.emplace_back(new CMSDY2D12Filter());
  target.emplace_back(new CMSJETS11Filter());
  target.emplace_back(new CMSWEASY840PBFilter());
  target.emplace_back(new CMSWMASY47FBFilter());
  target.emplace_back(new CMSWMU8TEVFilter());
  target.emplace_back(new CMSZDIFF12Filter());
  target.emplace_back(new CMS_1JET_8TEVFilter());
  target.emplace_back(new CMS_2JET_3D_8TEVFilter());
  target.emplace_back(new CMS_2JET_7TEVFilter());
  target.emplace_back(new CMS_HMDY_13TEVFilter());
  target.emplace_back(new CMS_HMDY_DE_13TEVFilter());
  target.emplace_back(new CMS_HMDY_DM_13TEVFilter());
  target.emplace_back(new CMS_ZCHARM_DIFF_UNNORM_8TEVFilter());

```

**Comment by Zaharid** [comment]
@enocera just to  say we are going to be looking into this data again.

**Comment by Zaharid** [comment]
Not that I know of, other than perhaps some rubber-stamping.

**Comment by enocera** [comment]
There seems to be a memory leak that requires investigation.

**Review by enocera — APPROVED** [review]
This is good to go. I've only fixed a memory leak.

**Pull request merged**